### ISSUE TITLE
Create Apache OpenWhisk mailing list page listing

### ISSUE THREAD

**Issue by jlengstorf** [openwhisk/openwhisk.github.io#32]
For community updates, discussion, and announcements.

**Comment by jlengstorf** [comment]
@krook I'm pretty sure you said this was in progress, so I'm dropping it in that column. Feel free to edit this, reassign it, or whatever else if I'm wrong.

**Comment by krook** [comment]
@jlengstorf @mrutkows was going to let us know when the lists were created by Apache, then we could link to them as Cordova does from one of our pages: https://cordova.apache.org/contact/

**Comment by krook** [comment]
This is also related to adding a contact address, such as help@openwhisk.org

**Comment by mrutkows** [comment]
I believe that our "contact us" model should mirror what Cordova does here:
[https://cordova.apache.org/contact/](https://cordova.apache.org/contact/)

Specifically, and since we have only the "dev" list for now, which would mean we would have: 

Development list:
- Writing to: dev@openwhisk.incubator.apache.org
- Subscribe to: dev-subscribe@@openwhisk.incubator.apache.org
- Unsubscribe to: dev-unsubscribe@@openwhisk.incubator.apache.org

-------------------------------------------------------------------------------
The full instructions are provided by the "bot": as follows: 

_To subscribe to the list, send a message to:
   <dev-subscribe@openwhisk.incubator.apache.org>

To remove your address from the list, send a message to:
   <dev-unsubscribe@openwhisk.incubator.apache.org>

Send mail to the following for info and FAQ for this list:
   <dev-info@openwhisk.incubator.apache.org>
   <dev-faq@openwhisk.incubator.apache.org>

Similar addresses exist for the digest list:
   <dev-digest-subscribe@openwhisk.incubator.apache.org>
   <dev-digest-unsubscribe@openwhisk.incubator.apache.org>

To get messages 123 through 145 (a maximum of 100 per request), mail:
   <dev-get.123_145@openwhisk.incubator.apache.org>

To get an index with subject and author for messages 123-456 , mail:
   <dev-index.123_456@openwhisk.incubator.apache.org>

They are always returned as sets of 100, max 2000 per request,
so you'll actually get 100-499.

To receive all messages with the same subject as message 12345,
send a short message to:
   <dev-thread.12345@openwhisk.incubator.apache.org>

The messages should contain one line or word of text to avoid being
treated as sp@m, but I will ignore their content.
Only the ADDRESS you send to is important.

You can start a subscription for an alternate address,
for example "john@host.domain", just add a hyphen and your
address (with '=' instead of '@') after the command word:
<dev-subscribe-john=host.domain@openwhisk.incubator.apache.org>

To stop subscription for this address, mail:
<dev-unsubscribe-john=host.domain@openwhisk.incubator.apache.org>

In both cases, I'll send a confirmation message to that address. When
you receive it, simply reply to it to complete your subscription._

**Comment by krook** [comment]
We should also sunset the Google mailing list as part of this: https://groups.google.com/forum/#!forum/openwhisk

**Comment by jlengstorf** [comment]
@mrutkows Okay — I just pushed a commit (15551dd) with the beginnings of the content for this page, but I need some help. What's the link for the online archive? I tried manipulating the Cordova archive links, but I wasn't able to figure out what our list is called.

Cordova link: <http://callback.markmail.org/search/?q=#query:%20list%3Aorg.apache.incubator.callback-dev+page:1+state:facets>

Or should I just not worry about this right now?
